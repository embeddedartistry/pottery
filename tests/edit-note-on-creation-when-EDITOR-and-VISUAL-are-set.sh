export EDITOR
export VISUAL

EDITOR=fake-editor
VISUAL=fake-visual

pottery init
pottery note
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
