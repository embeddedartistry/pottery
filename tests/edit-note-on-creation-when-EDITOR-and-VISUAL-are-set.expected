export EDITOR
export VISUAL

EDITOR=fake-editor
VISUAL=fake-visual

pottery init
pottery note
VISUAL docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
cat docs/project-history/1992/1992-01/1992-01-12T16:34:23Z_note_FAKEUNIQUEID.md
created by VISUAL
